dogpile/cache/region.py

```diff
@@ -1248,26 +1249,32 @@ def somemethod(self, x, y):
         if function_key_generator is None:
             function_key_generator = self.function_key_generator
 
-        def decorator(fn):
+        def get_or_create_wrapper(key_generator, user_func, *arg, **kw):
```

Comment:
I don't think `get_or_create_wrapper` is the best name, since it just makes me ask "why do we need to wrap get_or_create?" This isn't so much wrapping it as orchestrating a call to it, since it's doing real work and has a different signature.

I would consider something like `get_or_create_for_user_func` or just `get_or_create`, maybe, idk.

Reply:
agreed - not the best naming